zigpy/zcl/__init__.py

```diff
@@ -682,10 +682,10 @@ def write_attributes_undivided(
         args = self._write_attr_records(attributes)
         return self._write_attributes_undivided(args, manufacturer=manufacturer)
 
-    def bind(self):
+    async def bind(self):
         return self._endpoint.device.zdo.bind(cluster=self)
 
-    def unbind(self):
+    async def unbind(self):
         return self._endpoint.device.zdo.unbind(cluster=self)
```

Comment:
Here too:

```suggestion
        return await self._endpoint.device.zdo.unbind(cluster=self)
```

Reply:
Done in 8f64a19eaf061d0b02604735ecba4e29c6365769.